### ISSUE TITLE
Problem with database

### ISSUE THREAD

**Issue by haziqahbalqis** [Phype/telnet-iot-honeypot#18]
Can anyone know how to fix my problem? I have already get to connect with honeypot.py but when I link with telnet 127.0.0.1 2223 the backend.py cannot read the data.

Thank you

![backend py result](https://user-images.githubusercontent.com/46991089/51666371-a6111a80-1ff8-11e9-8a34-25056bd8f1d3.PNG)
![honeypot py result](https://user-images.githubusercontent.com/46991089/51666380-a90c0b00-1ff8-11e9-83b4-201b657f4ea8.PNG)
![telnet 127 0 0 1 2223](https://user-images.githubusercontent.com/46991089/51666387-ad382880-1ff8-11e9-8d77-8cd54ab04583.PNG)

**Comment by f1r3walled** [comment]
Hello,
Unhappy, I had no success while troubleshooting the project, like the last time. I tried a lot of possibilities and changes in the configuration files and packages installation, but none worked for me. 
The behavior is ever the same, it means, no data is recorded on the database and not even shown on the frontend.
I tried the manual installation and also the procedure through vagrant, but the behavior is the same for both situations.
My error message is referred to the following link "http://sqlalche.me/e/e3q8", but it is not telling enough to finish the troubleshooting.
Anyway, I imagine that this project has ended, because there is no updated since more than one year ago.
I'll fork it to work in an improved version.
If your project or curiosity is related to honeypot, I recommend you MHN (https://github.com/threatstream/mhn), which is a great project and largely supported by the community.

Cheers!
